nemo_skills/pipeline/openrlhf/ppo.py

```diff
@@ -89,7 +88,6 @@ def format_train_args(self):
             f" --max_ckpt_mem 10000000000 "
             f" --save_path {os.path.join(self.output_dir, 'checkpoints')} "
             f" --save_steps -1 "
-            # f" --max_samples 500000 "
```

Comment:
Shouldn't we keep a high value for Max samples?

Reply:
Not sure, this was removed in one of the other PRs, I just removed the comment. Feel free to add it back if it's needed